### PR TITLE
Update Default styles of Widgets

### PR DESCRIPTION
**Pull request by jsartisan**
## Description
Updating default styles to fix design inconsistency in widgets.

## Type of change
- Bug fix

## How Has This Been Tested?
- manually

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/components/designSystems/blueprint/DropdownComponent.tsx [hunk 1]
```diff
@@ -94,8 +94,10 @@ const StyledControlGroup = styled(ControlGroup)<{ haslabel: string }>`
 const DropdownStyles = createGlobalStyle`
   .select-popover-wrapper {
     width: 100%;
-    border-radius: ${(props) => props.theme.radii[1]}px;
-    box-shadow:  0px 2px 4px rgba(67, 70, 74, 0.14);
+    border: 1px solid #e1e1e1;
```
Comment (riodeuno) [inline app/client/src/components/designSystems/blueprint/DropdownComponent.tsx:97]: Is this the same as `props.theme.borders[2]`?

File: app/client/src/components/designSystems/blueprint/DropdownComponent.tsx [hunk 2]
```diff
@@ -94,8 +94,10 @@ const StyledControlGroup = styled(ControlGroup)<{ haslabel: string }>`
...
+    border-radius: ${(props) => props.theme.radii[0]}px;
```
Comment (riodeuno) [inline app/client/src/components/designSystems/blueprint/DropdownComponent.tsx:98]: We can just make this `0`. We won't be changing this anytime soon, and will be devising a new strategy for themes within canvas widgets,

File: app/client/src/components/designSystems/blueprint/DropdownComponent.tsx (hunk 1 above)
Comment (jsartisan) [reply app/client/src/components/designSystems/blueprint/DropdownComponent.tsx:97]: Oh right. I missed this one. Updating this.

**Review by riodeuno — DISMISSED** [review]
Please check the popovers across the application once. We've made the box-shadow in popovers !important in index.css. Just need to make sure, it doesn't effect the UI elsewhere.

**Pull request merged**